packages/grid/x-data-grid-generator/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-data-grid-generator",
-  "version": "6.0.4",
+  "version": "6.0.5",
```

AFAIK we always sync this package version with the `data-grid` itself 🤔 
```suggestion
  "version": "6.1.0",
```